website/settings/defaults.py

```diff
@@ -103,6 +103,14 @@ def parent_dir(path):
 ENABLE_EMAIL_SUBSCRIPTIONS = True
 MAILCHIMP_GENERAL_LIST = 'Open Science Framework General'
 
+#Triggered emails
+OSF_GENERAL_LIST = 'Open Science Framework Help'
```

Comment:
I think this should be renamed to something like `OSF_HELP_LIST` (feel free to give it a better name), since there is already a general mailing list.